interface/src/app/tribe_client/geneset-search-modal.tpl.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal-header">
+    <geneset-search-form organism="organism"><geneset-search-form>
```

Comment:
☆ ...and this is where you will need to bind the new `results` parameter (see `.directive`) to the parent's `scope`.